source/includes/connection-snippets/scram-snippets/scala-connection.scala

```diff
@@ -0,0 +1,8 @@
+// begin scram connection
+import org.mongodb.scala._
+
+val uri: String = "mongodb+srv://<username>:<password>@<cluster-address>/test?retryWrites=true&w=majority"
+System.setProperty("org.mongodb.async.type", "netty")
```

Comment:
Remove the system property - netty is no longer a requirement and system properties for configuration of the driver settings are being removed in the near future.

Reply:
Could use `streamType=netty` in the uri string.

Reply:
Fixed - removed system property & updated uri to streamType=netty